packages/api/src/votingStrategies/linearQuadraticFunding.ts

```diff
@@ -418,24 +418,37 @@ export const matchQFContributions = async (
   for (const contribution of contributions) {
     const { projectId, amount, token, contributor } = contribution;
 
-    if (!contributionTokens.includes(token)) {
```

Comment:
This seems redundant as this is already done in L404. 
This is unreachable code from what I can tell